scripts/deploy.ts

```diff
@@ -83,7 +86,8 @@ async function main() {
         default: 'https://metadata.sealcred.xyz',
       },
       incrementalBinaryTreeLibAddress: {
```

Comment:
Let's add a prompt asking if you want to transfer old accounts
It will look like this:
```TS
      shouldTransferOldAccounts: {
        type: 'boolean',
        required: true,
        default: true,
      },
```

Reply:
Don't forget to check for `if (shouldTransferOldAccounts)` below